fronted/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.2.0",
     "@testing-library/jest-dom": "^5.11.4",
```

Comment:
shouldn't `tailwindcss` appear in package too? I tried locally it's giving me this error `UnhandledPromiseRejectionWarning: Error: Cannot find module 'tailwindcss'` @ziadbaroudi do you think it's related.